e2e/pages/events/manageOrders.page.js

```diff
@@ -283,16 +283,28 @@ const selectCafcassRegion = region => {
   I.click(`${section4.cafcassRegion.group}-${region}`);
 };
 
-const selectEnglandOffice= office => {
+const selectEnglandOffice = office => {
   I.selectOption(section4.englandOffices, office);
 };
 
+const linkApplication = async (applicationToChoose) => {
+  I.see('Is there an application for the order on the system?');
+  I.dontSee('Applications');
+  I.checkOption('Yes', '#manageOrdersShouldLinkApplication');
+  I.see('Applications');
```

Comment:
Fair enough. I'll make the change.

Reply:
Actually, @Rob-Doherty. Thinking about it, this element shows up as soon as the element above it is clicked (i.e. no page is reloaded). Do you still think it's worth having a "waitForText()" in this case?

Reply:
I was thinking that even without the network calls it could just render slower since it'll be running in a VM for the nightly browsers, but there's a built-in 200ms delay in `stepListener.js` helper that I'd forgotten about so lets go with your original line and we can always change it later if it's actually needed